src/main/webapp/app/entities/study/study-detail.component.html

```diff
@@ -19,6 +19,10 @@ <h2>{{study.name}}</h2>
         <dd>
             <span>{{study.emailBody}}</span>
         </dd>
+        <dt><span>FAQ</span></dt>
+        <dd>
+            <span>{{study.faq}}</span>
+        </dd>
```

Comment:
Will all the additional information be hosted in this special study page, or will it link to a separate page?

Reply:
Definitely on a separate page as this special study page is only available to view to authenticated users. Study information is able to be viewed and edited by the user who created them.

Reply:
Great that sounds good